plugin/client_rpc.go

```diff
@@ -62,8 +62,9 @@ type apiRPCClient struct {
 }
 
 type apiRPCServer struct {
-	impl      API
-	muxBroker *plugin.MuxBroker
+	impl         API
+	muxBroker    *plugin.MuxBroker
+	apiRPCClient *apiRPCClient
```

Comment:
Is this every assigned somewhere?

Reply:
yes, on line 771!

Reply:
Hey @hanzei, wondering if you have got time to come back to this ? 😄

Reply:
In L 771 you access `apiRPCClient`. I was wondering where you _assign_ a value to it.

Reply:
Whoops, yeah. so I should assign it something like this :

```
s.apiRPCClient = &apiRPCClient{
		client:    rpc.NewClient(connection),
		muxBroker: s.muxBroker,
	}
```
in the function `InstallPlugin`? Does that sound right to you @hanzei ?

Reply:
After re-reading the code, I don't think that calling a log method on the client side is a good idea. if `s.muxBroker.Dial` the RPC connection won't work and hence calling the client side logging method won't neither.

Either we should stick with `fmt.Fprintf` or just `mlog`

Reply:
Thanks. For now, i have kept it to `fmt.Fprintf`